packages/augur-ui/src/modules/events/actions/log-handlers.ts

```diff
@@ -21,7 +21,7 @@ import { Action } from 'redux';
 import { AppState } from 'store';
 import { updateBlockchain } from 'modules/app/actions/update-blockchain';
 import { isSameAddress } from 'utils/isSameAddress';
-import { Events, Logs, TXEventName } from '@augurproject/sdk';
+import { Events, Logs, TXEventName, NULL_ADDRESS } from '@augurproject/sdk';
```

Comment:
is NULL_ADDRESS used anywhere?

Reply:
It was used in my initial PR changes. Good catch, will remove.